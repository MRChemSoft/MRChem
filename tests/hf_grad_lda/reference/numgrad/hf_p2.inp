world_prec = 1.0e-6                 # Overall relative precision
world_size = 6                      # Size of simulation box 2^n
world_unit = angstrom

# use Psi4 naming of displacements
# displacement 5 of 5 (i.e. point +2 in the stencil) 
Molecule {
$coords
F           -1.194726305678    -0.244580636233     0.340708526570
H           -1.675185087490    -0.395926884536     1.111606880519
$end
translate = false
}

WaveFunction {
  method = LDA                      # Wave function method (HF or DFT)
}

SCF {
  guess_type = CORE_DZ             # Type of initial guess: none, mw, gto
}
