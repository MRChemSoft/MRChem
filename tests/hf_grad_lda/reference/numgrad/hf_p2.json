{
  "input": {
    "molecule": {
      "charge": 0,
      "coords": [
        {
          "atom": "f",
          "xyz": [
            -2.2577055116173503,
            -0.4621904178670923,
            0.6438458035420823
          ]
        },
        {
          "atom": "h",
          "xyz": [
            -3.1656410234133596,
            -0.7481933771493701,
            2.1006325624305116
          ]
        }
      ],
      "multiplicity": 1
    },
    "mpi": {
      "bank_size": -1,
      "numerically_exact": false,
      "shared_memory_size": 10000
    },
    "mra": {
      "basis_order": 9,
      "basis_type": "interpolating",
      "boxes": [
        2,
        2,
        2
      ],
      "corner": [
        -1,
        -1,
        -1
      ],
      "max_scale": 20,
      "min_scale": -6
    },
    "printer": {
      "file_name": "hf_p2.inp",
      "print_level": 0,
      "print_mpi": false,
      "print_prec": 6,
      "print_width": 75
    },
    "rsp_calculations": {},
    "scf_calculation": {
      "fock_operator": {
        "coulomb_operator": {
          "poisson_prec": 1e-06,
          "shared_memory": false
        },
        "exchange_operator": {
          "exchange_prec": -1.0,
          "poisson_prec": 1e-06
        },
        "kinetic_operator": {
          "derivative": "abgv_55"
        },
        "nuclear_operator": {
          "proj_prec": 1e-06,
          "shared_memory": false,
          "smooth_prec": 1e-06
        },
        "xc_operator": {
          "shared_memory": false,
          "xc_functional": {
            "cutoff": 0.0,
            "functionals": [
              {
                "coef": 1.0,
                "name": "svwn5"
              }
            ],
            "spin": false
          }
        }
      },
      "initial_guess": {
        "file_basis": "initial_guess/mrchem.bas",
        "file_chk": "checkpoint/phi_scf",
        "file_gto_a": "initial_guess/mrchem.moa",
        "file_gto_b": "initial_guess/mrchem.mob",
        "file_gto_p": "initial_guess/mrchem.mop",
        "file_phi_a": "initial_guess/phi_a_scf",
        "file_phi_b": "initial_guess/phi_b_scf",
        "file_phi_p": "initial_guess/phi_p_scf",
        "localize": false,
        "method": "DFT (SVWN5)",
        "prec": 0.001,
        "restricted": true,
        "type": "core",
        "zeta": 2
      },
      "properties": {
        "dipole_moment": {
          "dip-1": {
            "operator": "h_e_dip",
            "precision": 1e-06,
            "r_O": [
              0.0,
              0.0,
              0.0
            ]
          }
        }
      },
      "scf_solver": {
        "checkpoint": false,
        "energy_thrs": -1.0,
        "file_chk": "checkpoint/phi_scf",
        "final_prec": 1e-06,
        "helmholtz_prec": -1.0,
        "kain": 5,
        "localize": false,
        "max_iter": 100,
        "method": "DFT (SVWN5)",
        "orbital_thrs": 9.999999999999999e-06,
        "rotation": 0,
        "start_prec": 1e-06
      }
    },
    "schema_name": "mrchem_input",
    "schema_version": 1
  },
  "output": {
    "properties": {
      "center_of_mass": [
        -2.3034482298448204,
        -0.47659953900917984,
        0.7172402080604454
      ],
      "charge": 0,
      "dipole_moment": {
        "dip-1": {
          "magnitude": 0.7087062385487939,
          "r_O": [
            0.0,
            0.0,
            0.0
          ],
          "vector": [
            -0.3697567904862602,
            -0.11647491191165038,
            0.5932773747097162
          ],
          "vector_el": [
            23.115233837433603,
            4.791432226084335,
            -7.30196741962523
          ],
          "vector_nuc": [
            -23.484990627919863,
            -4.907907137995985,
            7.895244794334946
          ]
        }
      },
      "geometric_derivative": {
        "electronic": [
          0.0,
          0.0,
          0.0
        ],
        "electronic_norm": 0.0,
        "nuclear": [
          0.0,
          0.0,
          0.0
        ],
        "nuclear_norm": 0.0,
        "total": [
          0.0,
          0.0,
          0.0
        ],
        "total_norm": 0.0
      },
      "geometry": [
        {
          "symbol": "F",
          "xyz": [
            -2.2577055116173503,
            -0.4621904178670923,
            0.6438458035420823
          ]
        },
        {
          "symbol": "H",
          "xyz": [
            -3.1656410234133596,
            -0.7481933771493701,
            2.1006325624305116
          ]
        }
      ],
      "multiplicity": 1,
      "orbital_energies": {
        "energy": [
          -24.08772032874935,
          -1.0958452082658627,
          -0.49828474704555376,
          -0.3605550021818303,
          -0.3605549746820689
        ],
        "occupation": [
          2.0,
          2.0,
          2.0,
          2.0,
          2.0
        ],
        "spin": [
          "p",
          "p",
          "p",
          "p",
          "p"
        ],
        "sum_occupied": -52.80592052184933
      },
      "scf_energy": {
        "E_ee": 55.44417795002923,
        "E_eext": 0.0,
        "E_el": -105.02381001423025,
        "E_en": -249.74536814355378,
        "E_kin": 99.4147485312776,
        "E_next": 0.0,
        "E_nn": 5.171754257836299,
        "E_nuc": 5.171754257836299,
        "E_tot": -99.85205575639395,
        "E_x": 0.0,
        "E_xc": -10.137368351983312
      }
    },
    "provenance": {
      "creator": "MRChem",
      "mpi_processes": 1,
      "nthreads": 20,
      "routine": "mrchem.x",
      "total_cores": 20,
      "version": "1.1.0-alpha"
    },
    "rsp_calculations": null,
    "scf_calculation": {
      "initial_energy": {
        "E_ee": 94.58966787598048,
        "E_eext": 0.0,
        "E_el": -83.20021086194238,
        "E_en": -329.23944111555113,
        "E_kin": 166.756883963381,
        "E_next": 0.0,
        "E_nn": 5.171754257836299,
        "E_nuc": 5.171754257836299,
        "E_tot": -78.02845660410608,
        "E_x": 0.0,
        "E_xc": -15.30732158575273
      },
      "scf_solver": {
        "converged": true,
        "cycles": [
          {
            "energy_terms": {
              "E_ee": 58.07762744731867,
              "E_eext": 0.0,
              "E_el": -104.40083150463325,
              "E_en": -247.19752732188644,
              "E_kin": 95.01710809472922,
              "E_next": 0.0,
              "E_nn": 5.171754257836299,
              "E_nuc": 5.171754257836299,
              "E_tot": -99.22907724679695,
              "E_x": 0.0,
              "E_xc": -10.298039724794696
            },
            "energy_total": -99.22907724679695,
            "energy_update": 21.200620642690865,
            "mo_residual": 1.1964281956165346,
            "wall_time": 24.888084892
          },
          {
            "energy_terms": {
              "E_ee": 50.70359621258219,
              "E_eext": 0.0,
              "E_el": -104.70600524990125,
              "E_en": -239.22508118716374,
              "E_kin": 93.31052706330037,
              "E_next": 0.0,
              "E_nn": 5.171754257836299,
              "E_nuc": 5.171754257836299,
              "E_tot": -99.53425099206495,
              "E_x": 0.0,
              "E_xc": -9.495047338620092
            },
            "energy_total": -99.53425099206495,
            "energy_update": 0.3051737452680072,
            "mo_residual": 0.5063379574793708,
            "wall_time": 22.587878761
          },
          {
            "energy_terms": {
              "E_ee": 57.25317303668711,
              "E_eext": 0.0,
              "E_el": -104.96709047491203,
              "E_en": -251.94569203672592,
              "E_kin": 100.0610062873545,
              "E_next": 0.0,
              "E_nn": 5.171754257836299,
              "E_nuc": 5.171754257836299,
              "E_tot": -99.79533621707573,
              "E_x": 0.0,
              "E_xc": -10.335577762227727
            },
            "energy_total": -99.79533621707573,
            "energy_update": 0.261085225010774,
            "mo_residual": 0.3691604003996881,
            "wall_time": 23.70011245
          },
          {
            "energy_terms": {
              "E_ee": 54.90527848638512,
              "E_eext": 0.0,
              "E_el": -105.00817641183372,
              "E_en": -248.28523869494188,
              "E_kin": 98.42491512351828,
              "E_next": 0.0,
              "E_nn": 5.171754257836299,
              "E_nuc": 5.171754257836299,
              "E_tot": -99.83642215399742,
              "E_x": 0.0,
              "E_xc": -10.053131326795237
            },
            "energy_total": -99.83642215399742,
            "energy_update": 0.04108593692168938,
            "mo_residual": 0.12900565767707756,
            "wall_time": 22.791857465
          },
          {
            "energy_terms": {
              "E_ee": 55.75289667440907,
              "E_eext": 0.0,
              "E_el": -105.02153477385372,
              "E_en": -250.34063515317547,
              "E_kin": 99.74067668134047,
              "E_next": 0.0,
              "E_nn": 5.171754257836299,
              "E_nuc": 5.171754257836299,
              "E_tot": -99.84978051601742,
              "E_x": 0.0,
              "E_xc": -10.174472976427777
            },
            "energy_total": -99.84978051601742,
            "energy_update": 0.013358362019999959,
            "mo_residual": 0.07954856250778669,
            "wall_time": 23.877126403
          },
          {
            "energy_terms": {
              "E_ee": 55.41857903660777,
              "E_eext": 0.0,
              "E_el": -105.02351159645832,
              "E_en": -249.7199076036878,
              "E_kin": 99.41134317443243,
              "E_next": 0.0,
              "E_nn": 5.171754257836299,
              "E_nuc": 5.171754257836299,
              "E_tot": -99.85175733862202,
              "E_x": 0.0,
              "E_xc": -10.133526203810723
            },
            "energy_total": -99.85175733862202,
            "energy_update": 0.001976822604603967,
            "mo_residual": 0.022366460840096428,
            "wall_time": 24.619726285
          },
          {
            "energy_terms": {
              "E_ee": 55.441411571619824,
              "E_eext": 0.0,
              "E_el": -105.02379223960456,
              "E_en": -249.7529842728848,
              "E_kin": 99.42502185347995,
              "E_next": 0.0,
              "E_nn": 5.171754257836299,
              "E_nuc": 5.171754257836299,
              "E_tot": -99.85203798176826,
              "E_x": 0.0,
              "E_xc": -10.137241391819533
            },
            "energy_total": -99.85203798176826,
            "energy_update": 0.0002806431462403225,
            "mo_residual": 0.009724675773673105,
            "wall_time": 24.226435975
          },
          {
            "energy_terms": {
              "E_ee": 55.44612346442926,
              "E_eext": 0.0,
              "E_el": -105.02380679266162,
              "E_en": -249.75176077904695,
              "E_kin": 99.41949037692143,
              "E_next": 0.0,
              "E_nn": 5.171754257836299,
              "E_nuc": 5.171754257836299,
              "E_tot": -99.85205253482532,
              "E_x": 0.0,
              "E_xc": -10.137659854965356
            },
            "energy_total": -99.85205253482532,
            "energy_update": 1.4553057056332364e-05,
            "mo_residual": 0.0012234844515445536,
            "wall_time": 23.867952476
          },
          {
            "energy_terms": {
              "E_ee": 55.444291206950275,
              "E_eext": 0.0,
              "E_el": -105.02380975131284,
              "E_en": -249.74315496756014,
              "E_kin": 99.41239937830264,
              "E_next": 0.0,
              "E_nn": 5.171754257836299,
              "E_nuc": 5.171754257836299,
              "E_tot": -99.85205549347654,
              "E_x": 0.0,
              "E_xc": -10.137345369005615
            },
            "energy_total": -99.85205549347654,
            "energy_update": 2.9586512226842387e-06,
            "mo_residual": 0.0007423567218074579,
            "wall_time": 23.197679274
          },
          {
            "energy_terms": {
              "E_ee": 55.44399887990491,
              "E_eext": 0.0,
              "E_el": -105.02380994252373,
              "E_en": -249.74365034015193,
              "E_kin": 99.4131662086623,
              "E_next": 0.0,
              "E_nn": 5.171754257836299,
              "E_nuc": 5.171754257836299,
              "E_tot": -99.85205568468743,
              "E_x": 0.0,
              "E_xc": -10.137324690938996
            },
            "energy_total": -99.85205568468743,
            "energy_update": 1.9121088712381606e-07,
            "mo_residual": 0.00018083338884155906,
            "wall_time": 23.774881171
          },
          {
            "energy_terms": {
              "E_ee": 55.444229555307395,
              "E_eext": 0.0,
              "E_el": -105.0238100087851,
              "E_en": -249.74567319135747,
              "E_kin": 99.41501509134194,
              "E_next": 0.0,
              "E_nn": 5.171754257836299,
              "E_nuc": 5.171754257836299,
              "E_tot": -99.85205575094881,
              "E_x": 0.0,
              "E_xc": -10.137381464077
            },
            "energy_total": -99.85205575094881,
            "energy_update": 6.626137860621384e-08,
            "mo_residual": 0.00010181039865411604,
            "wall_time": 24.996294169
          },
          {
            "energy_terms": {
              "E_ee": 55.44414419717846,
              "E_eext": 0.0,
              "E_el": -105.02381001297361,
              "E_en": -249.7455567296259,
              "E_kin": 99.41497285115506,
              "E_next": 0.0,
              "E_nn": 5.171754257836299,
              "E_nuc": 5.171754257836299,
              "E_tot": -99.85205575513731,
              "E_x": 0.0,
              "E_xc": -10.137370331681236
            },
            "energy_total": -99.85205575513731,
            "energy_update": 4.1885073187586386e-09,
            "mo_residual": 2.2795196258568588e-05,
            "wall_time": 24.528034091
          },
          {
            "energy_terms": {
              "E_ee": 55.444228783272145,
              "E_eext": 0.0,
              "E_el": -105.02381001393086,
              "E_en": -249.74558067668278,
              "E_kin": 99.41491946068727,
              "E_next": 0.0,
              "E_nn": 5.171754257836299,
              "E_nuc": 5.171754257836299,
              "E_tot": -99.85205575609456,
              "E_x": 0.0,
              "E_xc": -10.13737758120748
            },
            "energy_total": -99.85205575609456,
            "energy_update": 9.57243173616007e-10,
            "mo_residual": 1.6972426137087215e-05,
            "wall_time": 24.197851635
          },
          {
            "energy_terms": {
              "E_ee": 55.44417795002923,
              "E_eext": 0.0,
              "E_el": -105.02381001423025,
              "E_en": -249.74536814355378,
              "E_kin": 99.4147485312776,
              "E_next": 0.0,
              "E_nn": 5.171754257836299,
              "E_nuc": 5.171754257836299,
              "E_tot": -99.85205575639395,
              "E_x": 0.0,
              "E_xc": -10.137368351983312
            },
            "energy_total": -99.85205575639395,
            "energy_update": 2.993942871398758e-10,
            "mo_residual": 9.920710996770436e-06,
            "wall_time": 23.682119955
          }
        ],
        "wall_time": 334.939402269
      },
      "success": true
    },
    "schema_name": "mrchem_output",
    "schema_version": 1,
    "success": true
  }
}
