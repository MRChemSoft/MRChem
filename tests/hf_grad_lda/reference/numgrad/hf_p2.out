                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.1.0-alpha                              ***
***                                                                     ***
***         Git branch         mrchem-gradient                          ***
***         Git commit hash    c6b193dd4223e5895077                     ***
***         Git commit author  Roberto Di Remigio                       ***
***         Git commit date    Fri Sep 25 22:32:15 2020 +0200           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :       (no bank)                               1
 OpenMP threads          :                                              20
 Total cores             :                                              20
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.3.1
 Git branch            : mrchem-gradient
 Git commit hash       : b79bdf06c3c82a53edb6
 Git commit author     : Roberto Di Remigio
 Git commit date       : Tue Sep 15 18:31:50 2020 +0200

 Linear algebra        : EIGEN v3.3.7
 Parallelization       : MPI/OpenMP

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 9
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   64.00000    64.00000    64.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -64.00000   -64.00000   -64.00000 ]
 upper bounds          : [   64.00000    64.00000    64.00000 ]
 total length          : [  128.00000   128.00000   128.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       F            :       -2.257706       -0.462190        0.643846
    1       H            :       -3.165641       -0.748193        2.100633
---------------------------------------------------------------------------
 Center of mass          :       -2.303448       -0.476600        0.717240
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                 Computing Ground State Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize Core Hamiltonian matrix
 Precision               : 1.00000e-03
 Restricted              : True
 AO basis                : Hydrogenic orbitals
 Zeta quality            : 2
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                               5
 Beta electrons          :                                               5
 Total electrons         :                                              10
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    2    p          :                              9.999932443729e-01
    1    2    p          :                              9.999510746687e-01
    2    2    p          :                              9.999999998876e-01
    3    2    p          :                              9.999999965148e-01
    4    2    p          :                              9.999999875184e-01
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (SVWN5)
 Precision               : 1.00000e-03
 Localization            : Off
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                166.756883963381
 E-N energy              :            (au)               -329.239441115551
 Coulomb energy          :            (au)                 94.589667875980
 Exchange energy         :            (au)                  0.000000000000
 X-C energy              :            (au)                -15.307321585753
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                  5.171754257836
 Ext. field (nuc)        :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)                -83.200210861942
 Nuclear energy          :            (au)                  5.171754257836
---------------------------------------------------------------------------
 Total energy            :            (au)             -7.802845660411e+01
                         :      (kcal/mol)             -4.896359537053e+04
                         :        (kJ/mol)             -2.048636831633e+05
                         :            (eV)             -2.123262377512e+03
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    2    p          :            (au)                -14.600146474836
    1    2    p          :            (au)                  3.229378313382
    2    2    p          :            (au)                  4.751600904833
    3    2    p          :            (au)                  4.776601960882
    4    2    p          :            (au)                  5.082841965388
---------------------------------------------------------------------------
 Sum occupied            :            (au)                  6.480553339298
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize ground state orbitals
 Method                  : DFT (SVWN5)
 Checkpointing           : Off
 Max iterations          : 100
 KAIN solver             : 5
 Localization            : Off
 Diagonalization         : First two iterations
 Start precision         : 1.00000e-06
 Final precision         : 1.00000e-06
 Helmholtz precision     : Dynamic
 Energy threshold        : Off
 Orbital threshold       : 1.00000e-05
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual             Total energy                Update
---------------------------------------------------------------------------
    0          2.236068e+00         -78.028456604106         -7.802846e+01
    1          1.196428e+00         -99.229077246797         -2.120062e+01
    2          5.063380e-01         -99.534250992065         -3.051737e-01
    3          3.691604e-01         -99.795336217076         -2.610852e-01
    4          1.290057e-01         -99.836422153997         -4.108594e-02
    5          7.954856e-02         -99.849780516017         -1.335836e-02
    6          2.236646e-02         -99.851757338622         -1.976823e-03
    7          9.724676e-03         -99.852037981768         -2.806431e-04
    8          1.223484e-03         -99.852052534825         -1.455306e-05
    9          7.423567e-04         -99.852055493477         -2.958651e-06
   10          1.808334e-04         -99.852055684687         -1.912109e-07
   11          1.018104e-04         -99.852055750949         -6.626138e-08
   12          2.279520e-05         -99.852055755137         -4.188507e-09
   13          1.697243e-05         -99.852055756095         -9.572432e-10
   14          9.920711e-06         -99.852055756394         -2.993943e-10
---------------------------------------------------------------------------
                      SCF converged in 14 iterations!
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       F            :       -2.257706       -0.462190        0.643846
    1       H            :       -3.165641       -0.748193        2.100633
---------------------------------------------------------------------------
 Center of mass          :       -2.303448       -0.476600        0.717240
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                 99.414748531278
 E-N energy              :            (au)               -249.745368143554
 Coulomb energy          :            (au)                 55.444177950029
 Exchange energy         :            (au)                  0.000000000000
 X-C energy              :            (au)                -10.137368351983
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                  5.171754257836
 Ext. field (nuc)        :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -105.023810014230
 Nuclear energy          :            (au)                  5.171754257836
---------------------------------------------------------------------------
 Total energy            :            (au)             -9.985205575639e+01
                         :      (kcal/mol)             -6.265811048625e+04
                         :        (kJ/mol)             -2.621615344446e+05
                         :            (eV)             -2.717112737221e+03
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    2    p          :            (au)                -24.087720328749
    1    2    p          :            (au)                 -1.095845208266
    2    2    p          :            (au)                 -0.498284747046
    3    2    p          :            (au)                 -0.360555002182
    4    2    p          :            (au)                 -0.360554974682
---------------------------------------------------------------------------
 Sum occupied            :            (au)                -52.805920521849
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :       23.115234        4.791432       -7.301967
 Magnitude               :            (au)                       24.710131
                         :         (Debye)                       62.806883
---------------------------------------------------------------------------
 Nuclear vector          :      -23.484991       -4.907907        7.895245
 Magnitude               :            (au)                       25.258013
                         :         (Debye)                       64.199460
---------------------------------------------------------------------------
 Total vector            :       -0.369757       -0.116475        0.593277
 Magnitude               :            (au)                        0.708706
                         :         (Debye)                        1.801351
===========================================================================


===========================================================================
                           Geometric Derivative
---------------------------------------------------------------------------
 Total                   :        0.000000        0.000000        0.000000
 Norm                    :            (au)                        0.000000
---------------------------------------------------------------------------
 Nuclear                 :        0.000000        0.000000        0.000000
 Norm                    :            (au)                        0.000000
---------------------------------------------------------------------------
 Electronic              :        0.000000        0.000000        0.000000
 Norm                    :            (au)                        0.000000
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h  5m 41s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
